code/modules/mob/living/simple_animal/friendly/adhomai.dm

```diff
@@ -1,6 +1,7 @@
 /mob/living/simple_animal/ice_tunneler
 	name = "ice tunneler"
 	desc = "An egg producing beast from Adhomai. It is known for burrowing in ice and snow."
+	icon = 'icons/mob/npc/livestock.dmi'
```

Comment:
This is messy. There are so many aniamals which can be united under `/livestock/` object class and have this assigned, to have cleaner code... but idk